src/main/kotlin/no/nav/syfo/dialogmotekandidat/domain/DialogmotekandidatEndring.kt

```diff
@@ -13,6 +13,7 @@ import java.util.*
 enum class DialogmotekandidatEndringArsak {
     STOPPUNKT,
     DIALOGMOTE_FERDIGSTILT,
+    DIALOGMOTE_LUKKET,
     UNNTAK,
     LUKKET,
```

Comment:
Hva er forskjellen på `LUKKET` og `DIALOGMOTE_LUKKET`?
Jeg er ikke helt inne i domenet for denne appen, dessverre :/

Reply:
Ja okey, skjønner! Manuelt lukket kandidat, og lukket pga manuelt lukket dialogmøte altså 😵‍💫